app/assets/stylesheets/master.scss

```diff
@@ -236,4 +235,18 @@ p {
 
 .modal-backdrop {
   z-index: 1030;
-}
\ No newline at end of file
+}
+
+.dashboard{
+  background-color: #D1D1D1;
+  min-height: 200px;
+  padding: 30px;
+}
+.user-photo{
+  border-radius: 50%;
+  border: 5px solid #AAAAAA;
+  height: 150px;
+  width: 150px;
+  background:url(trump.jpg) center;
```

Comment:
I'm confused..I have the adding the user photo/avatar issue but it looks like you're already starting stuff on it, @cacauzen?  Please clarify because I'd love to get some assistance via pair programming on this but do the coding majority of on my own so I can continue to learn.  Let me know if I have it wrong though.